library/xml-model/src/main/java/org/cru/godtools/xml/model/Text.kt

```diff
@@ -8,6 +8,7 @@ import org.ccci.gto.android.common.util.xmlpull.skipTag
 import org.cru.godtools.xml.XMLNS_CONTENT
 import org.xmlpull.v1.XmlPullParser
 
+private const val XML_RESOURCE = "resource"
```

Comment:
so, this is the incorrect attribute name for parsing `start-image` and `end-image` (or whatever I actually named the attributes)